frontend/src/lib/i18n/locales/en.json

```
@@ -575,6 +575,10 @@ If you don't see a dialog or already closed it, click the button below:",
     "create_project_request_email": {
       "subject": "Project request: {projectName}",
       "heading": "User {name} ({email}) requested that a project be created for them. Details below:"
+    },
+    "approve_project_request_email": {
+      "subject": "Project approval: {projectName}",
+      "heading": "{projectName} has been approved."
```

I'd go with something like this:
```suggestion
      "heading": "The project you requested, {projectName}, has been approved and created."
```

There should also be a link to the created project. Two ideas:
1) make the project name a link using markdown. There are a few examples in our translations e.g. `help_create_single_guest_user`. You'd need to then use a `<Markdown>` element inside `<mj-text>`. We've never done that before in an email, but I assume it should just work.
2) or add a button below the heading e.g. "View {projectName}". A lot of our emails have a button at the end.

---

I like having a button :)